README.md

```diff
@@ -192,6 +193,64 @@ When the SchemaTester loads a schema, it runs it through
 specification compliance issues. In case of issues with the schema itself, the validator will raise the appropriate
 error.
 
+## Django testing client
+
+The library includes an `OpenAPIClient`, which extends Django REST framework's
+[`APIClient` class](https://www.django-rest-framework.org/api-guide/testing/#apiclient).
+If you wish to validate each response against OpenAPI schema when writing
+unit tests - `OpenAPIClient` is what you need!
+
+To use `OpenAPIClient` simply pass `SchemaTester` instance that should be used
+to validate responses and then use it like regular Django testing client:
+
+```python
+schema_tester = SchemaTester()
+client = OpenAPIClient(schema_tester=schema_tester)
+response = client.get('/api/v1/tests/123/')
+```
+
+To force all developers working on the project to use `OpenAPIClient` simply
+override the `client` fixture (when using `pytest` with `pytest-django`):
+
+```python
+from pytest_django.lazy_django import skip_if_no_django
+
+from openapi_tester.schema_tester import SchemaTester
+
+
+@pytest.fixture
+def schema_tester():
+    return SchemaTester()
+
+
+@pytest.fixture
+def client(schema_tester):
+    skip_if_no_django()
+
+    from openapi_tester.clients import OpenAPIClient
+
+    return OpenAPIClient(schema_tester=schema_tester)
+```
+
+If you are using plain Django test framework, we suggest to create custom
+test case implementation:
+
+```python
+import functools
+
+from django.test.testcases import SimpleTestCase
+from openapi_tester.schema_tester import SchemaTester
+
+schema_tester = SchemaTester()
+
+
+class MyTestCase(SimpleTestCase):
+    client_class = functools.partial(OpenAPIClient, schema_tester=schema_tester)
```

Comment:
when an idea from comment https://github.com/snok/drf-openapi-tester/pull/262/#discussion_r851448465 will be implemented then this could be simplified to:

```suggestion
from django.test.testcases import SimpleTestCase
from openapi_tester.clients import OpenAPIClient


class MyTestCase(SimpleTestCase):
    client_class = OpenAPIClient
```

Reply:
You could also have 

```
schema_tester = SchemaTester()

class SimpleTestBase(SimpleTestCase):
    client_class = functools.partial(OpenAPIClient, schema_tester=schema_tester)
```

in some test file and just use that instead of `SimpleTestCase` right? That way there's no extra code needed for adding it to your test classes.

Reply:
Exactly, I will add some info that such subclass should be used instead of the original `SimpleTestCase` class